src/assets/stylesheet/styles.scss

```diff
@@ -0,0 +1 @@
+@import "../../../node_modules/govuk-frontend/govuk/all";
```

Comment:
@wilsond-gds I wasn't getting any styling locally because this was been gitignored for some reason so I pushed a commit to sort that out - hope you don't mind! Obvi feel free to change if you had other ideas.